.github/workflows/Unit-Test.yml

```diff
@@ -66,6 +74,11 @@ jobs:
             esac
       - name: Run unit tests
         run: |
+             if [[ -z ${NON_PKG_CHANGED_FILES} ]] && [[ $PLATFORM == 'linux/arm/v7' ]] && [ -n "${ARMV7L_PACKAGES+1}" ]; then
```

Comment:
Won't this condition always be true?  `[ -n "${ARMV7L_PACKAGES+1}" ]`

Reply:
From https://linuxsimply.com/bash-scripting-tutorial/conditional-statements/if/set-variable-if-not/

```
Similarly, you can employ the parameter expansion with the -n option to check if a variable is set. In this case, use the syntax: if [[ -n "${variable+value}" ]] where “-n” returns true only when the variable is set.
```

Reply:
Changing the logic...